esmvalcore/cmor/_fixes/emac/emac.py

```diff
@@ -211,6 +211,30 @@ def fix_metadata(self, cubes):
         return CubeList([cube])
 
 
+class Prodlnox(EmacFix):
+    """Fixes for ``prodlnox``."""
+
+    def fix_metadata(self, cubes):
+        """Fix metadata."""
+        cube = ((
+            self.get_cube(cubes, var_name=['NOxcg_cav', 'NOxcg_ave',
+                                           'NOxcg']).collapsed(
+                                               ['longitude', 'latitude'],
+                                               iris.analysis.SUM,
+                                               weights=None) +
```

Comment:
Just to make sure I'm getting this right: this variable is simply the total mass of that quantity in the grid cell, that's why you don't need weights for the sum?